MinecraftClient/Protocol/Handlers/Protocol18.cs

```diff
@@ -510,10 +509,10 @@ internal bool HandlePacket(int packetID, List<byte> packetData)
                         }
                         break;
                     case PacketIncomingType.WindowItems:
-                        if (handler.GetInventoryEnabled()||true)
+                        if (handler.GetInventoryEnabled()||true) // set true in .ini but GetInventoryEnabled seems like return false
```

Comment:
You need to adjust the check at line 70, otherwise inventory handling may auto-disable on server join:
```csharp
if (handler.GetInventoryEnabled() && protocolversion > MC114Version)
{
    ConsoleIO.WriteLineFormatted("§8Inventories are currently not handled for that MC version.");
    handler.SetInventoryEnabled(false);
}
```
This check sets the maximum version where inventory handling has been implemented and tested. On new Minecraft version, basic support is implemented first, and terrain may come at a later time. It might also be the case for inventories.